ProcessLib/SmallDeformation/Tests.cmake

```diff
@@ -84,6 +84,21 @@ AddTest(
     expected_cube_1e0_simple_shear_pcs_0_ts_4_t_1.000000.vtu cube_1e0_simple_shear_pcs_0_ts_4_t_1.000000.vtu NodalForces NodalForces
 )
 
+# Mechanics; Small deformations, linear (SDL); Material forces
+AddTest(
+    NAME Mechanics_SDL_material_forces_bar
+    PATH Mechanics/Linear/MaterialForces
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS bar.prj
+    WRAPPER time
+    TESTER vtkdiff
+    REQUIREMENTS NOT OGS_USE_MPI
+    ABSTOL 1e-14 RELTOL 1e-15
+    DIFF_DATA
+    expected_bar_out_pcs_0_ts_2_t_1.000000.vtu bar_out_pcs_0_ts_2_t_1.000000.vtu displacement displacement
+    expected_bar_out_pcs_0_ts_2_t_1.000000.vtu bar_out_pcs_0_ts_2_t_1.000000.vtu MaterialForces MaterialForces
+    expected_bar_out_pcs_0_ts_2_t_1.000000.vtu bar_out_pcs_0_ts_2_t_1.000000.vtu free_energy_density free_energy_density
+)
 
```

Comment:
I'd to see both the 3D and the 2D version of the code tested. Test case can be the same (bar) with the beam's axis lying in a different coordinate plane in the 3D case compared to the 2D case. Simply because we manually coded the Eshelby stress component-wise separately for 3D and 2D, and the tests should ensure functionality of both implementations.